README.md

```diff
@@ -2,13 +2,13 @@ shavar-list-creation
 ====================
     usage: lists2safebrowsing.py
 
-A shavar.ini file must exist in the same directory that specifies the following parameters. See sample_shavar_list_creation.ini for examples.
+A `shavar_list_creation.ini` file must exist in the same directory that specifies the following parameters. (See `sample_shavar_list_creation.ini` for examples.)
 
-Commits to shavar-list-exceptions trigger rebuilds of shavar-list-creation like so:
+Commits to [shavar-list-exceptions](https://github.com/mozilla-services/shavar-list-exceptions) trigger rebuilds of [shavar-list-creation](https://github.com/mozilla-services/shavar-list-creation) like so:
 
-1. A commit is made to https://github.com/mozilla-services/shavar-list-exceptions
+1. A commit is made to [shavar-list-exceptions](https://github.com/mozilla-services/shavar-list-exceptions)
```

Comment:
This is not the only trigger. Please add the time-based trigger as well because we'll be removing the `shavar-list-exceptions` as part of https://bugzilla.mozilla.org/show_bug.cgi?id=1277446.